src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +109,96 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
-Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
-you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+{{site.alert.note}}
+When you change the Flutter plugin dependencies in `my_flutter/pubspec.yaml`,
+run `flutter pub get` in your Flutter module directory to refresh the list
 of plugins read by the `podhelper.rb` script. Then run `pod install` again from
-`some/path/MyApp`.
+in your application at`some/path/MyApp`.
+{{site.alert.end}}
+
+The `podhelper.rb` script embeds your plugins, `Flutter.framework`, and
+`App.framework` into your project.
+
+You should now be able to build the project using `⌘B`.
+
+### Embed frameworks in Xcode
+
+Alternatively, you can generate the necessary frameworks and embed them in your application
+by manually editing your existing Xcode project. You may choose to do this if members of your
+team cannot locally install Flutter SDK and CocoaPods, or if you do not wish to use CocoaPods
+as a dependency manager in your existing applications.
+
+If you are using the above [Embed with CocoaPods and Flutter tools](#embed-with-CocoaPods-and-Flutter-tools)
+method, you can skip these instructions.
+
+The following example assumes you want to generate the frameworks to `some/path/MyApp/Flutter/`.
+
+```text
+some/path/
+  MyApp/
+    Flutter/
+      Debug/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework (each plugin with iOS platform code is a separate framework)
+      Profile/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework
+      Release/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework
+```
+
+```terminal
+$ flutter build ios-framework --output=some/path/MyApp/Flutter/
+```
 
-The `podhelper.rb` script ensures that your plugins, the Flutter.framework, and
-the App.framework are embedded in your project.
+{{site.alert.tip}}
+With Xcode 11 installed you can generate [XCFrameworks][] instead of universal frameworks by adding
+the flags `--xcframework --no-universal`.
+{{site.alert.end}}
+
+Open your existing project in Xcode and drag the frameworks from `some/path/MyApp/Flutter/Release/` in Finder
```

Comment:
Say that this is just one arbitrary way of integrating we're using as an example. You don't have to do this and you can integrate in whichever way you prefer.